tpcx_bb/xbb_tools/cluster_startup.py

```diff
@@ -84,8 +99,11 @@ def maybe_create_worker_directories(dask_worker):
     if create_blazing_context:
         bc = BlazingContext(
             dask_client=client,
-            pool=True,
+            pool=os.environ.get("BLAZING_POOL", False),
             network_interface=os.environ.get("INTERFACE", "eth0"),
```

Comment:
Should this be `ib0` as that's what we set in the bsql_startup script?

Everything else looks fine to me.

Reply:
Fixed, @VibhuJawa thanks for reviewing.

Reply:
Thanks for fixing it. Approving the PR :-D